torch/_guards.py

```diff
@@ -175,9 +175,12 @@ class Guard:
 
     stack = None
     user_stack = None
+    _hash = None
```

Comment:
Is it possible to make Guard a frozen dataclass so that the hash doesn't change?

Reply:
Not trivially since we modify the stack later on.